test/integration/test_download.py

```diff
@@ -45,19 +45,22 @@ def setup_class(self):
         self.branch = 'OMERO-DEV-latest'
         self.ice = '3.6'
 
+    @pytest.mark.skipif(True, reason='URL to be updated')
```

Comment:
What does "URL to be updated" mean?

Reply:
I meant: Change URL since it uses the old ci